src/content/components/text-input/usage.mdx

```diff
@@ -15,11 +30,39 @@ _Text inputs_ enable the user to interact with and input data. Use when the appl
 
 ## Format
 
+### Default values
+
+Where possible, add programmatic assistance. Detect and pre-fill inputs to reduce errors and save time. When the software can't determine the value that belongs in an input, use type-ahead to make suggestions. Use sentence-case for default values, detected values, and auto-completion text.
+
+### Validation and errors
+
+Real time validation helps to streamline the process and keep data clean when the user is filling out forms. For full guidelines, refer to the [forms](/components/form/usage) usage page.
+
+<ImageComponent cols="8" caption="Error example">
+
+![Validation text example](images/text-input-usage-2.png)
+
+</ImageComponent>
+
+### Character count
+
+Text fields and text areas may have a character limit and counter. Use the character limit and counter to let users know how long their entry can be. 
+
+<ImageComponent cols="8" caption="An example of a text area.">
+
+![An example of a text area.](images/text-area-usage-1.png)
+
+</ImageComponent>
+
+While useful for data collection, character count is not an intuitive way to communicate content length to users. Character count allows a user to clearly understand an input error if they surpass the character limit.
```

Comment:
I think it's better to keep all this content above the image, so it's less broken up.

Reply:
Sure thing. Changed!